pyevmasm/evmasm.py

```diff
@@ -624,22 +627,32 @@ def disassemble_one(bytecode, pc=0):
 
     assert isinstance(opcode, int)
 
-    instruction = instruction_table[opcode]
+
+    if silent:
+        if opcode not in instruction_table:
+            instruction = copy(instruction_table[opcode])
```

Comment:
Should this just get instruction_table[0xfe] for INVALID? Also is there something in the yellow paper that describes the behavior when an unimplemented instruction is run? Just want to make sure it's semantically the same as INVALID, but I don't see anything.

Reply:
Opps I ment to get 0xfe.

Unimplemented instruction is defined in the yellow (if I remember correctly) as the same as an INVALID. Only diff is that it may get redefined I guess ..

Reply:
```
    try:
        instruction = copy(instruction_table[opcode])
    except KeyError:
        if silent:
            instruction = copy(instruction_table[0xfe])
        else:
            raise
```
 ??